### PR TITLE
updated config of hleix and kalseed filters and sequences

### PR DESCRIPTION
**Pull request by gianipez**
- HelixFilter and KalSeedFilter have a new configuration that includes n>= 1 set of cuts
- The number of track-based sequences is halved: [ DeM, DeP] -> De

### CODE REVIEW

File: python/generateMenuFromJSON.py [hunk 1]
```diff
@@ -44,13 +48,21 @@ def generateLogger(evtMode, outdir, dictLog, logName, dictStreams, isOfflineBuil
...
-            loggerConfig.write('      maxSubRuns   : 1\n')
-            loggerConfig.write('      fileName     : "{}.art\"\n'.format(k))
-            loggerConfig.write('   }\n\n')
+            loggerConfig.write('         {}:'.format(streamName)+' { \n')
+            loggerConfig.write('            module_type: RootDAQOutput \n')
+            loggerConfig.write('            SelectEvents : {}\n'.format(json.dumps(dictStreams[k])))
+            loggerConfig.write('            maxSubRuns   : 1\n')
+            loggerConfig.write('            fileName     : "{}_%r_%#.art\"\n'.format(k))
+            loggerConfig.write('         }\n\n')
+    if doIt == True:
```
Comment (brownd1978) [inline python/generateMenuFromJSON.py:57]: I don't know enough python to understand why you split the following logic into 3 stanzas.   It looks odd but not broken.

**Review by brownd1978 — APPROVED** [review]

**Comment by kutschke** [comment]
Merging now and will merge Offline 1238 shortly after.

**Comment by gianipez** [comment]
Thanks! As the trigger paths changed the trigger validation will fail the first day

**Comment by kutschke** [comment]
> Thanks! As the trigger paths changed the trigger validation will fail the first day

Thanks for the heads up.  I will comment tomorrow morning after the report appear in email.

**Pull request merged**